src/bridging/providers/across/util.ts

```diff
@@ -0,0 +1,69 @@
+import { TargetChainId } from '../../../chains'
+import { SuggestedFeesResponse } from './AcrossApi'
+import { AcrossQuoteResult } from './AcrossBridgeProvider'
+import { AcrossChainConfig, ACROSS_TOKEN_MAPPING } from './const/tokens'
+
+export function getChainConfigs(
+  sourceChainId: TargetChainId,
+  targetChainId: TargetChainId
+): { sourceChainConfig: AcrossChainConfig; targetChainConfig: AcrossChainConfig } | undefined {
+  const sourceChainConfig = getChainConfig(sourceChainId)
+  const targetChainConfig = getChainConfig(targetChainId)
+
+  if (!sourceChainConfig || !targetChainConfig) return
+
+  return { sourceChainConfig, targetChainConfig }
+}
+
+function getChainConfig(chainId: number): AcrossChainConfig | undefined {
+  return Object.values(ACROSS_TOKEN_MAPPING).find((config) => config.chainId === chainId)
+}
+
+export function getTokenSymbol(tokenAddress: string, chainConfig: AcrossChainConfig): string | undefined {
+  return Object.keys(chainConfig.tokens).find((key) => chainConfig.tokens[key] === tokenAddress)
+}
+
+export function getTokenAddress(tokenSymbol: string, chainConfig: AcrossChainConfig): string | undefined {
+  return chainConfig.tokens[tokenSymbol]
```

Comment:
What if there are several tokens with the same symbol?

Reply:
I don't expect this implementation to be the final one. I just want to allow to bridge to a handful of tokens and networks. I don't think the future provider will be having all the routes in a map (probably will use the API).

Reply:
So, we won't have duplicates for this controlled set of tokens.